libbeat/common/docker/client_test.go

```diff
@@ -53,7 +53,7 @@ func TestNewClient(t *testing.T) {
 	// Test we can hardcode version
 	os.Setenv("DOCKER_API_VERSION", "1.22")
 
-	client, err = NewClient(host, nil, nil)
+	client, err = NewClient("", nil, nil)
```

Comment:
Why is this change needed?

Reply:
Made the change to give coverage to the fallback, as this test creates 2 clients, the first part of the test sets host this part will not.